Makefile

```diff
@@ -75,22 +75,14 @@ clean:
 
 binaries: critest crictl
 
-install-critest:
-	install -D -m 755 $(CURDIR)/_output/critest$(BIN_EXT) $(BINDIR)/critest$(BIN_EXT)
+install-critest: critest
```

Comment:
Can you try to drop the dependency:
```suggestion
install-critest:
```

Reply:
I don't have better ideas either, let's drop the dependency of critest/crictl

Reply:
fixed by #689